{
  "comments": [
    {
      "key": {
        "uuid": "009d98a6_012bfb48",
        "filename": "tests/test_zuul_scheduler.py",
        "patchSetId": 2
      },
      "lineNbr": 767,
      "author": {
        "id": 24
      },
      "writtenOn": "2019-05-07T08:29:21Z",
      "side": 1,
      "message": "Which cover the bug we had yesterday. The test fail with:\n\n  Independent pipeline \u0027test\u0027 must accept a change without any approval\n  Independent pipeline \u0027test-prio\u0027 must accept a change without any approval",
      "range": {
        "startLine": 767,
        "startChar": 20,
        "endLine": 767,
        "endChar": 66
      },
      "revId": "36b5b4679c19fc1b23d87b1b529931877963ce1b",
      "serverId": "e9e9afe9-4712-486d-8885-f54b72dd1951",
      "unresolved": false
    }
  ]
}